includes/class-indieauth-authorization-endpoint.php

```diff
@@ -218,7 +272,7 @@ public function authorize() {
 		$client_icon = $info->get_icon();
 		$redirect_uri  = isset( $_GET['redirect_to'] ) ? wp_unslash( $_GET['redirect_to'] ) : null;
 		$scope         = isset( $_GET['scope'] ) ? wp_unslash( $_GET['scope'] ) : null;
-		$scopes        = explode( ' ', $scope );
+		$scopes        = array_filter( explode( ' ', $scope ) );
 		$state         = isset( $_GET['state'] ) ? $_GET['state'] : null;
 		$me            = isset( $_GET['me'] ) ? wp_unslash( $_GET['me'] ) : null;
```

Comment:
Actually, it should remove me entirely and instead use the user's URL. If you check the spec changes, the me sent in is a hint. So, since we are always using the logged in user, it should be replaced with either the root domain or /author/username as outlined elsewhere in the code

Reply:
There was never any danger of spoofing another URL. But the new version won't allow you to spoof the displayed version.